multiversxetl/indexer.py

```diff
@@ -1,33 +1,35 @@
-from typing import Any, Dict, Iterable
+from typing import Any, Dict, Iterable, Optional
 
 import elasticsearch.helpers
 from elasticsearch import Elasticsearch
 
+from multiversxetl.constants import (ELASTICSEARCH_MAX_RETRIES,
+                                     ELASTICSEARCH_MAX_SIZE)
+
 SCROLL_CONSISTENCY_TIME = "10m"
-SCAN_BATCH_SIZE = 9999
+SCAN_BATCH_SIZE = 7500
```

Comment:
We had a couple of "Scroll request has only succeeded on n shards out of m" errors. I thought that this might help - I am not sure if it did, but that error didn't happen for some days now.